tests/testthat/test-translate_sql.R

```diff
@@ -25,7 +25,7 @@ with_locale(test.locale(), test_that)('as() works', {
   )
   expect_equal(
     translate_sql(pmax(x), con=s[['con']]),
-    dplyr::sql('GREATEST("x")')
```

Comment:
Seems like the new version of dbplyr does not auto-uppercase the function translations. Could you instead modify R/sql_translate_env.R to return all uppercase function names?

Reply:
Done